schema/aws/defaults/infrastructure/virtual-machine.yml

```
@@ -14,7 +14,7 @@ specification:
   tags:
     - version: 0.4.2
   size: t2.micro
-  os_full_name: "ubuntu/images/hvm-ssd/ubuntu-bionic-18.04-amd64-server-20210907"
+  os_full_name: ubuntu/images/hvm-ssd/ubuntu-focal-20.04-amd64-server-20211129
```

```suggestion
  os_full_name: ubuntu/images/hvm-ssd/ubuntu-focal-20.04-amd64-server-20220104
```
New images have just been published, let's use them :)

---

Image updated